atlite/convert.py

```diff
@@ -128,22 +129,56 @@ def convert_and_aggregate(
             res = da.sum("time", keep_attrs=True)
             return maybe_progressbar(res, show_progress, **dask_kwargs)
 
+    if matrix is not None:
+
+        if shapes is not None:
+            raise ValueError(
+                "Passing matrix and shapes is ambiguous. Pass " "only one of them."
+            )
+
+        if isinstance(matrix, xr.DataArray):
+
+            if {"x", "y"}.issubset(matrix.dims):
+                matrix = matrix.stack(spatial=["y", "x"])
```

Comment:
Based on which convention are we stacking here? If this is following a convention already followed somewhere else in `atlite`, maybe we can not hardcode it but reference the appropriate object and follow its stacking rule?